addons/dotnet/dotnet.addon

```diff
@@ -0,0 +1,8 @@
+# Name: dotnet
+# Description: ImageStreams and Templates for developing and deploying application using .NET Core
```

Comment:
You need to add some more details to this addon metadata like we have for xpaas addon.

-> https://github.com/minishift/minishift/blob/master/addons/xpaas/xpaas.addon#L3

Also in the commit message make sure you mention which commit hash you picked.

-> https://github.com/minishift/minishift/commit/d7f374aaa585bf9b967631683f030f809ad0a0ea#diff-d9bde864a3a599589769c6f8cb32c9ab